src/api/res.py

```diff
@@ -115,7 +118,7 @@ def res(
 
     # Print automatically
     printable_result = PrintableResult(result)
-    if c.configuration.print_auto:
+    if (c.configuration.print_auto and print is not False) or print is True:
```

Comment:
- `if print is not False` -> `if print`
- if `print is True` -> `if print`

Reply:
ah I think you did this because of a conflict with the built-in print function, right?

Reply:
No, I did this, because `print` can take three values: `True`, `False` and `None`. The latter has a special role here. It being set means, that no overwrite of the default behavior is wished by the user.

Reply:
ahh I forgot that `print` can be `None` as well, thanks